cluster-provision/k8s/1.24/manifests/kubeadm.conf

```diff
@@ -28,20 +28,17 @@ apiServer:
     mountPath: /var/log/k8s-audit
     name: audit-log
   timeoutForControlPlane: 4m0s
-apiVersion: kubeadm.k8s.io/v1beta2
+apiVersion: kubeadm.k8s.io/v1beta3
 certificatesDir: /etc/kubernetes/pki
 clusterName: kubernetes
 controllerManager:
   extraArgs:
-    feature-gates: IPv6DualStack=true
     node-cidr-mask-size-ipv6: "116"
 dns:
   type: CoreDNS
 etcd:
   local:
     dataDir: /var/lib/etcd
-featureGates:
```

Comment:
Please include this change in the commit message. Something like "IPv6 Dual stack is default now".

Reply:
Added note to the following commit message - https://github.com/kubevirt/kubevirtci/pull/800/commits/809e0476f9061a2dc510553436f25b197efc9606